res/controllers/Numark-Party-Mix.midi.xml

```diff
@@ -317,21 +317,21 @@
           <script-binding />
         </options>
       </control>
-      <!-- Scratch toggle buttons (release) -->
+      <!-- Jogwheels -->
       <control>
         <group>[Channel1]</group>
-        <key>NumarkPartyMix.deck[0].scratchToggle.input</key>
-        <status>0x80</status>
-        <midino>0x07</midino>
+        <key>NumarkPartyMix.deck[0].wheelTurn.input</key>
+        <status>0xB0</status>
+        <midino>0x06</midino>
         <options>
           <script-binding />
         </options>
       </control>
       <control>
         <group>[Channel2]</group>
-        <key>NumarkPartyMix.deck[1].scratchToggle.input</key>
-        <status>0x81</status>
-        <midino>0x07</midino>
+        <key>NumarkPartyMix.deck[1].wheelTurn.input</key>
+        <status>0xB1</status>
+        <midino>0x06</midino>
```

Comment:
removing the NoteOff message handlers is not a good idea generally. Otherwises there might be some surprises when you change the handler later.

Reply:
Can you elaborate? My reasoning for keeping all the NoteOffs even if they seem unnecessary, is that the components handlers might depend on them. For buttons, you can specify `type: components.Button.prototype.types.toggle` (or something like that, I don't remember exactly).
The XML is not exactly a specification either, its more like a legacy artifact we aim to remove some time in the future.

Reply:
I will not change it.

Reply:
this device uses 0x9Y for NoteOn and 0x8Y for NoteOff messages so the unmapped values shouldn't cause trouble. i improved the scratchtoggle code

Reply:
the issue I see is that the unmapped values don't end up at the input handlers of the components, even though those components probably expect them. So it could cause weird behavior from the mapping code.